### ISSUE TITLE
pnext: Server log flooded with warnings

### ISSUE THREAD

**Issue by linkdata** [Starcounter/Starcounter#3307]
**@per-samuelsson reported https://github.com/Starcounter/Replicator/issues/9 but the issue is not related to Replicator specifically, but occurs when simply starting a database in pnext**

When starting a database the server log is flooded with warnings:

![image](https://cloud.githubusercontent.com/assets/2902045/12482684/4ddef382-c051-11e5-81f0-03465e726b0e.png)

As the warning level is to be used to inform the user about something he should immediately address, before it becomes an error, I'm not sure what I should do with the information I'm being given?

It's also hard to see the errors for all the warnings.

/CC @eriohl @bigwad

**Comment by k-rus** [comment]
This is due to #3271

**Comment by per-samuelsson** [comment]
> This is due to #3271

So the severity level is correct then? A **warning** is something you expect the user to address, and hopefully soon, or it will likely become an error. I couldn't really tell, but want to assure we follow the guidelines we have used historically:

```
Critical	Critical conditions
Error	Error conditions
Warning	May indicate that an error will occur if action is not taken.
Notice	Events that are unusual, but not error conditions.	
Informational	Normal operational messages that require no action.
Debug	Diagnostics
```

**Comment by k-rus** [comment]
Currently it is correct. It might be good idea to increase the severity after #3204 is fixed. Thus I will keep this issue open to do it.

**Comment by per-samuelsson** [comment]
Confused user: https://github.com/Starcounter/SweOffshoreIssues/issues/74

**Comment by k-rus** [comment]
I fixed issue #3204, so the amount of warnings is significantly reduced.

**Comment by miyconst** [comment]
@k-rus thank you, it was really annoying.

**Comment by per-samuelsson** [comment]
> I fixed issue #3204, so the amount of warnings is significantly reduced.

Great, @k-rus. So can we close this issue then?
